src/game/gameflow.c

```diff
@@ -1310,15 +1310,31 @@ GameFlow_InterpretSequence(int32_t level_num, GAMEFLOW_LEVEL_TYPE level_type)
             break;
 
         case GFS_LEVEL_STATS:
-            Stats_Show((int32_t)(intptr_t)seq->data);
+            PHASE_STATS_DATA phase_args = {
```

Comment:
Add parenthesis around this statement to fix compile fail.

Reply:
Addressed